placement/interface.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
hmm... I already have a placement.go that provides the basic structs for some of the interfaces